packages/coinstac-ui/app/render/components/user/form-login.jsx

```diff
@@ -83,6 +83,26 @@ class FormLogin extends Component {
     this.setState({ openSetStartupDirectoryDialog: false });
   }
 
+  renderError = () => {
+    const { auth, classes } = this.props;
+
+    if (!auth.error) {
+      return null;
+    }
+
+    const errorMessage =
+      auth.error === 'Expired token'
```

Comment:
String comparison like this is not really good... 2 options here. You could either turn this *Expired token* into an error code like *EXPIRED_TOKEN* and extract it into a constants file (maybe in the utils folder) - this way it's less error prone (you would be comparing with a constant). Or you could pass the error message "*Your login session...*" from the top...